alacritty.yml

```diff
@@ -211,39 +211,39 @@
 
   # Normal colors
   #normal:
-  #  black:   '#000000'
-  #  red:     '#d54e53'
-  #  green:   '#b9ca4a'
-  #  yellow:  '#e6c547'
-  #  blue:    '#7aa6da'
-  #  magenta: '#c397d8'
-  #  cyan:    '#70c0ba'
-  #  white:   '#eaeaea'
+  #  black:   '#1d1f21'
+  #  red:     '#cc6666'
+  #  green:   '#b5bd68'
+  #  yellow:  '#f0c674'
+  #  blue:    '#81a2be'
+  #  magenta: '#b294bb'
+  #  cyan:    '#8abeb7'
+  #  white:   '#c5c8c6'
 
   # Bright colors
   #bright:
   #  black:   '#666666'
-  #  red:     '#ff3334'
-  #  green:   '#9ec400'
+  #  red:     '#d54e53'
+  #  green:   '#b9ca4a'
   #  yellow:  '#e7c547'
   #  blue:    '#7aa6da'
-  #  magenta: '#b77ee0'
-  #  cyan:    '#54ced6'
-  #  white:   '#ffffff'
+  #  magenta: '#c397d8'
+  #  cyan:    '#70c0b1'
+  #  white:   '#eaeaea'
 
   # Dim colors
   #
   # If the dim colors are not set, they will be calculated automatically based
   # on the `normal` colors.
   #dim:
-  #  black:   '#000000'
-  #  red:     '#8c3336'
-  #  green:   '#7a8530'
-  #  yellow:  '#97822e'
-  #  blue:    '#506d8f'
-  #  magenta: '#80638e'
-  #  cyan:    '#497e7a'
-  #  white:   '#9a9a9a'
+  #  black:   '#232425'
+  #  red:     '#864343'
+  #  green:   '#777c44'
+  #  yellow:  '#9e824c'
+  #  blue:    '#556a7d'
+  #  magenta: '#75617b'
+  #  cyan:    '#5b7d78'
+  #  white:   '#828482'
```

Comment:
I'd assume these are just the automatically computed colors?

Reply:
Yes. These are normals * 0.66

Reply:
That doesn't seem to quite check out though. #1d1f21 * 0.66 should be less than those values. However #232425 is brighter in every single bit? So looks like something went wrong? Should be more around #131818 based on some rough estimation, though we should obviously pick the exact colors Alacritty computes by logging them and just copying them to the config file.

Reply:
That doesn't seem to quite check out though. #1d1f21 * 0.66 should be less than those values. However #232425 is brighter in every single bit? So looks like something went wrong? Should be more around #131818 based on some rough estimation, though we should obviously pick the exact colors Alacritty computes by logging them and just copying them to the config file.

Reply:
yep, thanks for spotting this out. One digit mistake copied three times. :(

Reply:
Double checked other dims, they seem to be correct